src/game/CowGame.tsx

```diff
@@ -19,27 +19,25 @@ export function CowGame() {
   // Hide box when pressing Space
   const hideMessageBox = () => setShowMessageBox(false);
 
-  // Run game when main frame is mounted
-  const runGameRef = useCallback(
-    (node) => {
-      if (node !== null) {
-        console.log("🐮🐮 Loading Cow Runner 🐮🐮");
+  useEffect(() => {
+    console.log("🐮🐮 Loading Cow Runner 🐮🐮");
 
-        // Boot runner
-        const runner = new Runner(".interstitial-wrapper", undefined);
-        console.log("Runner", runner);
+    // Boot runner
+    const runner = new Runner(".interstitial-wrapper", undefined);
+    console.log("Runner", runner);
```

Comment:
do we need/want these logs? maybe create a util function that only logs in develop